chef/data_bags/crowbar/template-network.json

```diff
@@ -87,6 +87,26 @@
             },
             "intf2": {
               "if_list": [ "1g1", "1g2" ]
+            },
+            "intf3": {
+              "if_list": [ "1g1", "1g2" ]
+            }
+          }
+        },
+        {
+          "pattern": "quad/.*/.*",
```

Comment:
I did some more testing and decided that it's easier to set ironic things up without this "quad" mode. Ironic network is needed on a limited set of nodes and in production setup you need to prepare custom network.json anyway. I will remove the "quad" option for now.